chains/rari/metadata.yaml

```diff
@@ -5,8 +5,8 @@ blockExplorers:
     url: https://mainnet.explorer.rarichain.org
 blocks:
   confirmations: 1
-  estimateBlockTime: 2
-  reorgPeriod: 1
+  estimateBlockTime: 30
+  reorgPeriod: 0
```

Comment:
why 0?

Reply:
observed long times between blocks at the times I checked